modules/shared/components/molecules/PostFooterCreation/PostFooterCreation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IPostFooterCreation } from './IPostFooterCreation';
+import styles from './PostFooterCreation.module.css';
+import Privacy from '../Privacy/Privacy';
+import Divider from '../../atoms/Divider/Divider';
+
+/** 
+* @Notice you will use the handleChange functions to get and use the value of the toggler and the select
+* @ThingsNeededToBeDoneInParentComponent
+const [togglerIsChecked,setTogglerIsChecked] = useState(false);
+const handleTogglerChange = (e: React.ChangeEvent<HTMLInputElement>):void => { setTogglerIsChecked(e.target.checked) }<Test>;
+const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>):void => {console.log(e.target.value)};
+*/
+const PostFooterCreation: FC<IPostFooterCreation.IProps> = ({
```

Comment:
Components names example  : <Button /> not <ButtonCreation />  and so on. 
I think no need to adding include `Creation` text in the component name.

Reply:
there are two-component as a footer 1- for the post view 2- for the post creation, so there is a component called post footer which is already existing so I named this one the post footer creation